server/test/helper/TestHelper.ts

```diff
@@ -39,4 +39,29 @@ export class TestHelper {
       return error;
     }
   }
+
+  public static getTokenForEmailAndPassword = async (
```

Comment:
is it possible to add this to the existing getToken method?

Reply:
Maybe. I didn't want to have a method with many parameter where some of them will never be used. If both is implemented in one function we always need to set some parameters to null?! I'm not sure what's the best way here. Should I try to combine these functions to one?

Reply:
no, its okay. now i know why you use the token query param. but please remove this query param.

Reply:
ok